### PR TITLE
Ramp fix

### LINKED ISSUE #928: Ramp exchanger namespace missing when installing samples
**Describe the bug**
Ramp exchanger namespace missing when installing samples.

**Steps**
1. Install samples.
2. See ramp exchanger error.

**Expected behavior**
Error shouldn't be there ideally.

**Additional context**
Moving the sample ramp script into the ramp package or adding compilation directives will fix the issue. I've updated the docs with a workaround for now so we're stable but this is something we should fix for 2.6.

### PR DESCRIPTION
**Pull request by sneakzttv**
Fixed ramp via conditional compilation to avoid changing the folder structure of samples. Closes #928

### CODE REVIEW

File: Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs:1]: No need to disable the entire script if ramp is not available.

**Review by rob1997 — APPROVED** [review]
Good quick fix

File: Packages/io.chainsafe.web3-unity.ramp/Samples~/Web3.Unity Ramp Samples/RampSample.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#define RAMP_AVAILABLE
```
[the remark this answers is not in the document]
Comment (rob1997) [reply Packages/io.chainsafe.web3-unity.ramp/Samples~/Web3.Unity Ramp Samples/RampSample.cs:1]: my bad wrong path `src/UnitySampleProject/Assets/Samples/web3.unity SDK Ramp/1.0.0/Web3.Unity Ramp Samples`

File: Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs
[the remark this answers is not in the document]
Comment (rob1997) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs:1]: my bad again wrong path `src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples`

[the remark this answers is not in the document]
Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs:1]: honestly completely forgot that we had 2 for a second there, fixed

File: Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs [hunk 2]
```diff
@@ -1,14 +1,15 @@
 using System;
 using System.Collections.Generic;
-using ChainSafe.Gaming.Exchangers.Ramp;
```
[the remark this answers is not in the document]
Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/DisableGameObjectIfServiceNotActive.cs:3]: Accessing the namespace directly supersedes it

File: Packages/io.chainsafe.web3-unity.ramp/Samples~/Web3.Unity Ramp Samples/RampSample.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity.ramp/Samples~/Web3.Unity Ramp Samples/RampSample.cs:1]: done

**Review by rob1997 — APPROVED** [review]
GGs

**Review by oleksandrchainsafe — APPROVED** [review]
👍👍👍

**Review by juans-chainsafe — APPROVED** [review]
Good work!

**Pull request merged**